javaagent-api/src/test/groovy/io/opentelemetry/javaagent/instrumentation/api/db/normalizer/SqlStatementInfoExtractorTest.groovy

```diff
@@ -0,0 +1,84 @@
+    'select col from (select * from anotherTable) alias'              | new SqlStatementInfo(sql, 'SELECT', null)
+    'select col from table1 union select col from table2'             | new SqlStatementInfo(sql, 'SELECT', null)
+    'select col from table where col in (select * from anotherTable)' | new SqlStatementInfo(sql, 'SELECT', null)
+    'select col from table1, table2'                                  | new SqlStatementInfo(sql, 'SELECT', null)
+    'select col from table1 t1, table2 t2'                            | new SqlStatementInfo(sql, 'SELECT', null)
+    'select col from table1 as t1, table2 as t2'                      | new SqlStatementInfo(sql, 'SELECT', null)
+    'select col from table where col in (1, 2, 3)'                    | new SqlStatementInfo(sql, 'SELECT', 'table')
+    'select col from table order by col, col2'                        | new SqlStatementInfo(sql, 'SELECT', 'table')
+    'select ąś∂ń© from źćļńĶ order by col, col2'                      | new SqlStatementInfo(sql, 'SELECT', 'źćļńĶ')
+    'select 12345678'                                                 | new SqlStatementInfo(sql, 'SELECT', null)
+    // Insert
+    ' insert into table where lalala'                                 | new SqlStatementInfo(sql.trim(), 'INSERT', 'table')
+    'insert insert into table where lalala'                           | new SqlStatementInfo(sql, 'INSERT', 'table')
+    'insert into db.table where lalala'                               | new SqlStatementInfo(sql, 'INSERT', 'db.table')
+    'insert without i-n-t-o'                                          | new SqlStatementInfo(sql, 'INSERT', null)
+    // Delete
+    'delete from table where something something'                     | new SqlStatementInfo(sql, 'DELETE', 'table')
+    'delete from 12345678'                                            | new SqlStatementInfo(sql, 'DELETE', null)
+    'delete   ((('                                                    | new SqlStatementInfo(sql, 'DELETE', null)
+    // Update
+    'update table set answer=42'                                      | new SqlStatementInfo(sql, 'UPDATE', 'table')
+    'update /*table'                                                  | new SqlStatementInfo(sql, 'UPDATE', null)
+    // Merge
+    'merge into table'                                                | new SqlStatementInfo(sql, 'MERGE', 'table')
+    'merge table (into is optional in some dbs)'                      | new SqlStatementInfo(sql, 'MERGE', 'table')
+    'merge (into )))'                                                 | new SqlStatementInfo(sql, 'MERGE', null)
+    // Unknown operation
+    'and now for something completely different'                      | new SqlStatementInfo(sql, null, null)
+    ''                                                                | new SqlStatementInfo(sql, null, null)
+    null                                                              | new SqlStatementInfo(sql, null, null)
+  }
```

Comment:
Maybe add a comment one just to document the known edge case (unless I'm misreading the code and this actually works)?
```
'/*comment*/ select * from table1' -> null/null
```

Reply:
This actually works, but I'll add a test case for that anyway.